### PR TITLE
update sql browser docs & screenshots

### LINKED ISSUE #65: Docs: Feedback for SQL Browser
Hi, I have some feedback about [this page](https://github.com/hazelcast/management-center-docs/edit/v/5.0-BETA-1/docs/modules/ROOT/pages/sql-browser.adoc)

In MC5.0, the SQL Browser is only available in the top nav window (and only available if connected to a 5.x cluster)

The docs say it is available in the left menu which is no longer the case

### PR DESCRIPTION
**Pull request by utkukaratas**
Fixes #65, #80

### CODE REVIEW

File: docs/modules/ROOT/pages/sql-browser.adoc [hunk 1]
```diff
@@ -14,38 +14,37 @@ image:ROOT:SQLBrowserWindow.png[SQLBrowserWindow]
...
 * **Export as CSV** button
 
-In the dropdown in the top left corner, you can choose one of the existing maps
-in the cluster, and it prefills the text area with the following query:
-`SELECT * FROM "<your-map-name>"`. You can execute the query by clicking
+The dropdown above lists your maps and mappings providing sample queries. If a map in
+your cluster does not have a mapping, selecting that map will insert a sample "create mapping"
+statement in the editor. If your map has a mapping, a "select" statement will be provided for you.
+
+You can execute the query by clicking
```
Comment (JakeSCahill) [inline docs/modules/ROOT/pages/sql-browser.adoc:25]:
```suggestion
To execute the autogenerated query, click
```

File: docs/modules/ROOT/pages/sql-browser.adoc [hunk 2]
```diff
@@ -14,38 +14,37 @@ image:ROOT:SQLBrowserWindow.png[SQLBrowserWindow]
...
 
-You can see the result of your query in the **Query Result** tab. You can click
+You can see the result of your query in the **Query Results** tab. You can click
 on the **Export as CVS** button to export the query result as a CSV file in
 a format https://tools.ietf.org/html/rfc4180[RFC 4180]
 
-image:ROOT:QueryResultTab.png[QueryResultTab]
-
-NOTE: Query result is limited to maximum of 1000 records.
+image:ROOT:SQLBrowserQueryResultTab.png[SQLBrowserQueryResultTab]
```
Comment (JakeSCahill) [inline docs/modules/ROOT/pages/sql-browser.adoc:37]:
```suggestion
image:ROOT:SQLBrowserQueryResultTab.png[A list of query results]
```

File: docs/modules/ROOT/pages/sql-browser.adoc [hunk 3]
```diff
@@ -14,38 +14,37 @@ image:ROOT:SQLBrowserWindow.png[SQLBrowserWindow]
...
-image:ROOT:JSONViewTab.png[JSONViewTab]
+image:ROOT:SQLBrowserJSONView.png[SQLBrowserJSONView]
 
 On the **History** tab, you can see the history of query execution. It holds last
 one hundred executed queries. Each row contains **Execution Time**,
 **Execution Status** and **Query** values. You can re-execute any query from the
 history by pushing a button in the **Execute** column.
 
-image:ROOT:HistoryTab.png[HistoryTab]
+image:ROOT:SQLBrowserHistoryTab.png[SQLBrowserHistoryTab]
```
Comment (JakeSCahill) [inline docs/modules/ROOT/pages/sql-browser.adoc:50]:
```suggestion
image:ROOT:SQLBrowserHistoryTab.png[A history of executed queries]
```

**Review by JakeSCahill — COMMENTED** [review]
Looks good. Just a few comments/suggestions.

I think the sentence about searching the dropdown should be moved to the paragraph where we introduce selecting map names from the dropdown.

Also, could we fix #80 in this PR too. Statements are documented here: https://docs.hazelcast.com/hazelcast/5.0-snapshot/sql/sql-statements.html

File: docs/modules/ROOT/pages/sql-browser.adoc [hunk 4]
```diff
@@ -14,38 +14,37 @@ image:ROOT:SQLBrowserWindow.png[SQLBrowserWindow]
...
 
-In the dropdown in the top left corner, you can choose one of the existing maps
-in the cluster, and it prefills the text area with the following query:
-`SELECT * FROM "<your-map-name>"`. You can execute the query by clicking
+The dropdown above lists your maps and mappings providing sample queries. If a map in
+your cluster does not have a mapping, selecting that map will insert a sample "create mapping"
+statement in the editor. If your map has a mapping, a "select" statement will be provided for you.
+
+You can execute the query by clicking
 **Execute Query** button. Dropdown component supports textual search.
```
[the remark this answers is not in the document]
Comment (utkukaratas) [reply docs/modules/ROOT/pages/sql-browser.adoc:26]: yeah, that line is too obvious a statement. deleted.

File: docs/modules/ROOT/pages/sql-browser.adoc [hunk 5]
```diff
@@ -4,7 +4,7 @@
 This chapter details how a Hazelcast cluster can be queried with SQL using Management Center.
 
 You can use the SQL browser for this purpose, which can be opened using
-the "SQL Browser" menu item found in the top meny.
+the "SQL Browser" menu item found in the top menu.
```
Comment (JakeSCahill) [inline docs/modules/ROOT/pages/sql-browser.adoc:7]:
```suggestion
the *SQL Browser* menu item found in the top menu.
```

We should be consistent about how we format UI elements.

File: docs/modules/ROOT/pages/sql-browser.adoc [hunk 6]
```diff
@@ -14,38 +14,37 @@ image:ROOT:SQLBrowserWindow.png[SQLBrowserWindow]
...
-* **JSON View** tab
+* **Query Results** tab
 * **History** tab
 * **Export as CSV** button
 
-In the dropdown in the top left corner, you can choose one of the existing maps
-in the cluster, and it prefills the text area with the following query:
-`SELECT * FROM "<your-map-name>"`. You can execute the query by clicking
+To query a map, select its name from the dropdown. If the map does not have a mapping, an autogenerated `CREATE MAPPING` statement will be inserted into the editor. If the map already has a mapping, an autogenerated `SELECT` statement will be inserted into the editor.
+your cluster does not have a mapping, selecting that map will insert an autogenerated `CREATE MAPPING`
```
Comment (JakeSCahill) [inline docs/modules/ROOT/pages/sql-browser.adoc:22]:
```suggestion
```

This text should be deleted.

File: docs/modules/ROOT/pages/sql-browser.adoc [hunk 7]
```diff
@@ -14,38 +14,37 @@ image:ROOT:SQLBrowserWindow.png[SQLBrowserWindow]
...
+statement into the editor. If your map already has a mapping, selecting that map will insert a `SELECT` statement into the editor.
```
Comment (JakeSCahill) [inline docs/modules/ROOT/pages/sql-browser.adoc:23]:
```suggestion
```

This text should be deleted.

**Review by JakeSCahill — APPROVED** [review]

**Pull request merged**